sos/report/plugins/openvswitch.py

```diff
@@ -79,17 +79,19 @@ def setup(self):
             # Capture DPDK and other parameters
             "ovs-vsctl -t 5 get Open_vSwitch . other_config",
             # Capture OVS list
-            "ovs-vsctl list Open_vSwitch",
+            "ovs-vsctl -t 5 list Open_vSwitch",
             # Capture OVS interface list
-            "ovs-vsctl list interface",
+            "ovs-vsctl -t 5 list interface",
             # Capture OVS detailed information from all the bridges
             "ovs-vsctl list bridge",
             # Capture DPDK queue to pmd mapping
             "ovs-appctl dpif-netdev/pmd-rxq-show",
             # Capture DPDK pmd stats
             "ovs-appctl dpif-netdev/pmd-stats-show",
             # Capture DPDK pmd performance counters
-            "ovs-appctl dpif-netdev/pmd-perf-show"
+            "ovs-appctl dpif-netdev/pmd-perf-show",
+            # Capture ofproto tunnel configs
+            "ovs-appctl ofproto/list-tunnels"
```

Comment:
minor: seems like this leaked into the wrong commit

Reply:
Yep, I've moved it to the datapath commit.